nginx-php5/Dockerfile

```diff
@@ -1,12 +1,11 @@
-FROM unit9/base:latest
+FROM unit9/nginx:latest
 
 MAINTAINER Artiom Vasiliev <artiom@unit9.com>
 
 COPY ["php-fpm.rc", "/etc/service/php-fpm/run"]
-COPY ["nginx.rc", "/etc/service/nginx/run"]
 
 RUN apt-get update && \
-    apt-get install -y nginx php5 php5-common php5-curl php5-fpm php5-gd \
+    apt-get install -y php5 php5-common php5-curl php5-fpm php5-gd \
         php5-mysql php5-xmlrpc php5-xcache php5-memcache && \
     rm -rf /var/cache/apt /var/lib/apt/lists
 
```

Comment:
Wait, I see the problem.

Everything is named nginx.conf, but ends up in various places that are not /etc/nginx/nginx.conf. Not good.

Some things are named nginx.run, or nginx.rc, or else, but they all end up in /etc/service/nginx/run. Not good.

Some things override other random things.

Let me fix ;)